### PR TITLE
Diagnostic Response

### PR DESCRIPTION
**Pull request by Nishkixs**
- Add solution branch answers to diagnostic

### CODE REVIEW

File: diagnostic.md [hunk 1]
```diff
@@ -19,36 +19,43 @@ this repository (`unix-cli-diagnostic`).
...
 Open up this file in Atom, and write your answers below (where indicated).
 
-1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
-
- <!-- Answer Starts Here -->
+1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you
+know?
 
- <!-- Answer Ends Here -->
+Absolute. It starts with / (meaning the root directory) and continues from
```
Comment (berziiii) [inline diagnostic.md:42]: Correct.

File: diagnostic.md [hunk 2]
```diff
@@ -19,36 +19,43 @@ this repository (`unix-cli-diagnostic`).
...
+there.
 
-2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
+2. Suppose that we're working on a project, and we want to use a font that's
+being hosted somewhere on the internet. Would we use an absolute or relative
+path to refer to it? Why?
 
- <!-- Answer Starts Here -->
+Absolute. If something is hosted on the internet, the only way to refer to it is
```
Comment (berziiii) [inline diagnostic.md:49]: Correct.

File: diagnostic.md [hunk 3]
```diff
@@ -19,36 +19,43 @@ this repository (`unix-cli-diagnostic`).
...
+with an absolute path.
 
- <!-- Answer Ends Here -->
+3. Now suppose that we have an image file living inside our project. Would we
+refer to it with an absolute or relative path? Why?
 
-3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
+Relative. These files should be included with our projects and be referenced in
```
Comment (berziiii) [inline diagnostic.md:55]: Correct.

**Comment by berziiii** [comment]
4 / 6. Great start but don't forget to read the whole diagnostic. It asked you to create a sub directory with a file and text within that file. That should have been commited and pushed as well.

**Comment by patrickthesailorman** [comment]
hows this?

**Pull request closed without merging**